tests/kubernetes_test.py

```diff
@@ -447,7 +447,7 @@ def test_handle_event_lost(mock_kubernetes_task):
         )
     )
 
-    assert mock_kubernetes_task.is_unknown
+    assert mock_kubernetes_task.exit_status == exitcode.EXIT_KUBERNETES_TASK_LOST
```

Comment:
You might want to keep the old assert as well, since we do still want to verify 'lost' k8s events lead to marking the task as `unknown`.

Reply:
sadly the way we're checking unknow is by checking exit code 

``` python
  def is_unknown(self):
      return self.exit_status is None
```

Reply:
Ohh interesting; i didn't realize assigning the LOST's an exit code changed this behavior, thanks. 

Did you happen to glance through all the other places where we use `is_unknown`/unknown to check this change won't change any expected behavior there?

Reply:
It should not be a problem since we're only marking lost jobs with the special lost exit code

Reply:
[is_broken()](https://sourcegraph.yelpcorp.com/Yelp/Tron@d51984ef27faf1d743881a8c578f02e1ddd60c72/-/blob/tron/core/actionrun.py?L724) uses the `is_unknown` property and it looks like there's some [jobrun](https://sourcegraph.yelpcorp.com/Yelp/Tron@d51984ef27faf1d743881a8c578f02e1ddd60c72/-/blob/tron/core/jobrun.py?L259) code that uses `is_broken` to gate starting new action runs - but i'm not sure if this change impacts that in a meaningful way yet